Apigee/SmartDocs/Revision.php

```diff
@@ -301,12 +306,15 @@ public function toArray($verbose = TRUE)
      *
      * @param OrgConfig $config
      * @param string $modelId
+     * @param string $modelName
      */
-    public function __construct(OrgConfig $config, $modelId)
+    public function __construct(OrgConfig $config, $modelId, $modelName = "")
     {
         $this->blankValues();
         $this->apiId = $modelId;
```

Comment:
It just reflects the poorly-named variable in the JSON payload coming back from the Modeling API.